docs/source/quickstart/index.rst

```diff
@@ -16,46 +16,60 @@ See also the readme in the GitHub repository.
 Install Cloud Custodian
 -----------------------
 
-Cloud Custodian is a Python application and supports Python 3 on Linux, MacOS and
-Windows.
+These instructions will install Cloud Custodian. Cloud Custodian is a Python
+application and supports Python 3 on Linux, MacOS and Windows. We recommend
+using Python 3.6 or higher.
 
-We recommend using Python 3.6 or higher.
+NOTE: Ensure you install the correct follow on package depending on the cloud
```

Comment:
`follow on` --> `follow-on`

Reply:
I would also include a brief description of why additional packages are needed for Azure and GCP and not AWS. Something along the lines of: 

```Cloud Custodian supports AWS by default, meaning that no additional packages are required to use Cloud Custodian with AWS. However, you will need additional packages for other cloud providers. You can find installation instructions for these packages below.```